kmesh_compile.sh

```diff
@@ -46,20 +46,25 @@ function copy_to_host() {
     else
         arch="aarch64"
     fi
+    
+    local kernel="normal"
```

Comment:
What kernel does normal kernel refer to, linux kernel 5.10 and above?
A comment could be added to clarify this.